eth_worker/sql_persistence/interface.py

```diff
@@ -452,4 +514,4 @@ def __init__(self, red, session, first_block_hash, PENDING_TRANSACTION_EXPIRY_SE
         
         self.first_block_hash = first_block_hash
 
-        self.PENDING_TRANSACTION_EXPIRY_SECONDS = PENDING_TRANSACTION_EXPIRY_SECONDS
\ No newline at end of file
+        self.PENDING_TRANSACTION_EXPIRY_SECONDS = PENDING_TRANSACTION_EXPIRY_SECONDS
```

Comment:
why CAPS?

Reply:
Because it's a [constant](https://www.python.org/dev/peps/pep-0008/#constants). FWIW, this line change was just adding a `\n` at the end of the file as well